cg-spec/editors_draft.html

```diff
@@ -657,15 +657,437 @@ <h2>Grammar</h2>
+            </div>
+            <div>
+              `|`
+              <code class="token">'isBLANK'</code>
+              <code class="token">'('</code>
+              <a data-cite="SPARQL11-QUERY#rExpression">Expression</a>
+              <code class="token">')'</code>
+            </div>
+            <div>
+              `|`
+              <code class="token">'isLITERAL'</code>
+              <code class="token">'('</code>
+              <a data-cite="SPARQL11-QUERY#rExpression">Expression</a>
+              <code class="token">')'</code>
+            </div>
+            <div>
+              `|`
+              <code class="token">'isNUMERIC'</code>
+              <code class="token">'('</code>
+              <a data-cite="SPARQL11-QUERY#rExpression">Expression</a>
+              <code class="token">')'</code>
+            </div>
+            <div>
+              `|`
+              <a data-cite="SPARQL11-QUERY#rRegexExpression">RegexExpression</a>
+            </div>
+            <div>
+              `|`
+              <a data-cite="SPARQL11-QUERY#rExistsFunc">ExistsFunc</a>
+            </div>
+            <div>
+              `|`
+              <a data-cite="SPARQL11-QUERY#rNotExistsFunc">NotExistsFunc</a>
+            </div>
+            <div style="font-weight: bold">
+              `|`
+              <code class="token">'TRIPLE'</code>
+              <code class="token">'('</code>
+              <a data-cite="SPARQL11-QUERY#rExpression">Expression</a>
+              <code class="token">','</code>
+              <a data-cite="SPARQL11-QUERY#rExpression">Expression</a>
+              <code class="token">','</code>
+              <a data-cite="SPARQL11-QUERY#rExpression">Expression</a>
+              <code class="token">')'</code>
+            </div>
+            <div style="font-weight: bold">
+              `|`
+              <code class="token">'SUBJECT'</code>
+              <code class="token">'('</code>
+              <a data-cite="SPARQL11-QUERY#rExpression">Expression</a>
+              <code class="token">')'</code>
+            </div>
+            <div style="font-weight: bold">
+              `|`
+              <code class="token">'PREDICATE'</code>
+              <code class="token">'('</code>
+              <a data-cite="SPARQL11-QUERY#rExpression">Expression</a>
+              <code class="token">')'</code>
+            </div>
+            <div style="font-weight: bold">
+              `|`
+              <code class="token">'OBJECT'</code>
+              <code class="token">'('</code>
+              <a data-cite="SPARQL11-QUERY#rExpression">Expression</a>
+              <code class="token">')'</code>
+            </div>
+            <div style="font-weight: bold">
+              `|`
+              <code class="token">'isTRIPLE'</code>
+              <code class="token">'('</code>
+              <a data-cite="SPARQL11-QUERY#rExpression">Expression</a>
+              <code class="token">')'</code>
+            </div>
```

Comment:
Do we need a clarifying note that, as with other SPARQL functions, they are case-insensitive? Or can this be safely assumed given the overall context?

Reply:
There's a blanket rule in SPARQL that all keywords are case-insensitive, that automatically covers this. I have no objection to a clarifying note but I don't think it's necessary.

Reply:
I agree with you and @jeenbroekstra. It doesn't hurt to have such a clarifying note. Added in commit https://github.com/w3c/rdf-star/pull/118/commits/d3570fcc59f4bcb4145c945aa928edb6d1233229